src/App.tsx

```diff
@@ -17,18 +76,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                searchQuery={searchQuery}
+                setSearchQuery={() => setSearchQuery}
```

Comment:
you can't pass functions for changing the state directly to the child component
let's create separate functions before the `return` statement, where we will accept the arguments and will pass them to the functions for changing the state

and will pass those wrappers to the child components